Csv to docs update (#123)
medic/medic-conf: README.md

To convert CSV to JSON docs, use the `csv-to-docs` action.

> ## csv file name

Reviewer: I think this should be a subheader of csv file name, right? Also, should have title case eg `### CSV File Name`

> ## csv file name

Reviewer: ```suggestion
### CSV File Name
```

> The name of the file drives what the `type` (person,place, report) and the value. Some examples shown below.

Reviewer: ```suggestion
  _The name of the file determines the type of doc created for rows contained in the file. The possible types are: `report`, `person`, and `place`. Each of these has a further specifier provided in the filename:_
- `place.{place_type}.csv`:  where `{place_type}` is the type of place specified in the file, one of: `clinic`, `health_center`, `district_hospital`
- `person.{parent_place_type}.csv`:  where `{parent_place_type}` is the parent for the person, one of: `clinic`, `health_center`, `district_hospital`
- `report.{form_id}.csv`:  where `{form_id}` is the form ID for all the reports in the file. You will need one file per form ID

Here are some examples:
```

File named place.district_hospital.csv  adds the property `"type":"district_hospital"`<br/>
File named person.clinic.csv add the property `"type":"person"`<br/>
File named report.immunization_visit.csv add the property `"type":"report", "form":"immunization_visit"`<br/>

> ## property types

Reviewer: ```suggestion
### Property Types
```

Reviewer: This should be done with a lot of different headers too.

## doc references

> In the reference example below. A `property` on the json doc will be populated with the `doc` that matches the `WHERE` statement.

Reviewer: Some of these words don't have to be shown as code.

```suggestion
In the reference example below. A property on the JSON doc will be populated with the doc that matches the `WHERE` statement.
```

> The CSV example below using the reference will find the `doc` with `district_1` and create a `parent` property with the value of the `district_1` doc

Reviewer: ```suggestion
The CSV example below using the reference will find the doc with `district_1` and create a `parent` property with the value of the `district_1` doc
```
